docs/features/private-protected.md

```diff
@@ -53,18 +53,18 @@ We modify the language specification as follows (additions in bold). Section num
 - A static class shall not include a sealed or abstract modifier. (However, since a static class cannot be instantiated or derived from, it behaves as if it was both sealed and abstract.)
 - A static class shall not include a class-base specification (§16.2.5) and cannot explicitly specify a base class or a list of implemented interfaces. A static class implicitly inherits from type object.
 - A static class shall only contain static members (§16.4.8). [Note: All constants and nested types are classified as static members. end note]
-- A static class shall not have members with protected**, private protected** or protected internal declared accessibility.
+- A static class shall not have members with protected **, private protected** or protected internal declared accessibility.
```

Comment:
Added missing comma. 
The order is different everywhere, which I agree is messy. I'll let Neal clean that up.